components/greaselion/browser/greaselion_service_impl.cc

```diff
@@ -282,29 +280,33 @@ void GreaselionServiceImpl::CreateAndInstallExtensions() {
         rule->has_unknown_preconditions() == false) {
       // Convert script file to component extension. This must run on extension
       // file task runner, which was passed in in the constructor.
+      GreaselionRule rule_copy(*rule);
```

Comment:
I guess it doesn't happen super frequently, but would be nice to estimate how large is the object and how often this copy would happen (to prevent any performance issues)

Reply:
Yeah, good point. Given our current `Greaselion.json`, each rule might be ~100 bytes or so on average and we currently have about 16 rules though not all of them are active typically. The copy would essentially occur on startup and on component update, so shouldn't be too frequent.